catalog/ui/catalog-ui-search/src/main/webapp/component/golden-layout/golden-layout.view.js

```diff
@@ -100,7 +102,7 @@ function getGoldenLayoutSettings(){
 // see https://github.com/deepstreamIO/golden-layout/issues/239 for details on why the setTimeout is necessary
 // The short answer is it mostly has to do with making sure these ComponentViews are able to function normally (set up events, etc.)
 function registerComponent(marionetteView, name, ComponentView) {
-    var options = marionetteView.options;
+    var options = _.extend({}, marionetteView.options, {lowBandwidth: router.get('lowBandwidth'), desiredContainer: name});
```

Comment:
There's no reason to pass the router value for lowBandiwdth in here when the LowBandwidthMapView can access it directly.

We should update the registerComponent function to take a fourth param called componentOptions and simply mix that in (rather than have desiredContainer always mixed in).

Reply:
Added that and an overloaded method with 3 parameters for backwards compatibility